packages/react/src/components/Slider/useSlider.ts

```diff
@@ -13,6 +13,8 @@ import {
 } from '@fluentui/utilities';
 
 export const ONKEYDOWN_TIMEOUT_DURATION = 1000;
+export const MIN_PREFIX = 'min';
+export const MAX_PREFIX = 'max';
```

Comment:
I don't think these need to be exported. In the test, just write out `min` and `max`. If they change then the test will fail, which is fine (the test can be updated).